features/0-install.feature

```diff
@@ -32,12 +32,7 @@ Feature: Install WordPress through the web UI
 
   @upstreamonly
   Scenario: Delete Akismet and Hello Dolly
-    When I go to "wp-login.php"
-    And I fill in "log" with "pantheon"
-    And I fill in "pwd" with "pantheon"
-    And I press "wp-submit"
-    Then print current URL
-    And I should be on "/wp-admin/"
+    Given I log in as an admin
```

Comment:
As we talked about yesterday, a better abstraction could go in WP-CLI instead. It'd like to merge this PR once the latest push passes.